templates/vue/src/components/tyde/TydeProgressBar.vue

```diff
@@ -0,0 +1,180 @@
+<template>
+  <div class="progress-wrapper">
+    <div class="bar-wrapper" :style="wrapperCSS">
+      <div class="outer-progress" :style="outerCSS">
+        <div class="inner-progress" :style="innerCSS"></div>
+        <img
+          v-for="n in incompleteStages"
+          :key="n"
+          :style="starStyle(n)"
+          :src="inactiveStarSrc"
+        />
+        <img
+          v-for="m in range(incompleteStages + 1, incompleteStages + completeStages)"
+          :key="m"
```

Comment:
Can we have more descriptive variables names? Not sure what `m` or `n` is supposed to represent here.